plugins/keyboard-navigation/src/navigation.js

```diff
@@ -502,12 +502,14 @@ export class Navigation {
     if (flyout && flyout.getWorkspace()) {
       const flyoutContents = flyout.getContents();
       const firstFlyoutItem = flyoutContents[0];
-      if (firstFlyoutItem instanceof Blockly.FlyoutButton) {
-        const astNode = Blockly.ASTNode.createButtonNode(firstFlyoutItem);
+      if (!firstFlyoutItem) return;
+      if (firstFlyoutItem.type === 'button' && firstFlyoutItem.button) {
```

Comment:
core uses an enum so yes it will be lowercase. the enum isn't exported so i can't use it here. it's probably fine to just check the property... we don't actually use the type property anywhere afaik. i don't know why we have it. will update

Reply:
Sweet feel free to merge after update! or if you decide to not change it. Lemme know if you'd prefer re-review.